LLVM_FOR_RISCV.md

```diff
@@ -40,100 +40,16 @@ successfully deactivated the environments.
   conda deactivate     # leave base
 ```
 
-# LLVM for Baremetal
+# Use script to build LLVM
 
-## Directory Setup for Build Files and Installation (Baremetal)
-
-Create a directory for RISC-V files and `_install` directory:
-
-```bash
-cd /data/users/$USER/condor # or your preferred workspace
-mkdir llvm-baremetal
-cd llvm-baremetal
-mkdir _install
-```
-
-## The RISC-V GNU Toolchain for Baremetal
-
-The `RISC-V GNU toolchain` is necessary to use the LLVM, you should be able to find it under `/data/tools/riscv64-unknown-elf`. Copy it to the `_install` directory in your workspace
+**Running the Script:**
 
 ```bash
-cp -fa /data/tools/riscv64-unknown-elf/* _install/
+cd $TOP
+bash how-to/scripts/build_llvm.sh
```

Comment:
These are the instructions, 'here run this script'?

this is a giant process, maybe a bit more explanation is in order.